packages/client/webapp/src/components/Sidebar.vue

```diff
@@ -50,49 +49,42 @@ function moveSidebar() {
 </script>
 
 <template>
-	<section :class="{ right: isSidebarPositionRight }">
-		<nav class="mode-selection">
-			<ul>
-				<li :active="selectedMode === Mode.SimulationPlan" @click="updateMode(Mode.SimulationPlan)">
-					<IconDataPlayer32 />
-				</li>
-				<li :active="selectedMode === Mode.ModelView" @click="updateMode(Mode.ModelView)">
-					<IconMachineLearningModel32 />
-				</li>
-				<li :active="selectedMode === Mode.Datasets" @click="updateMode(Mode.Datasets)">
-					<IconTableSplit32 />
-				</li>
-				<li :active="selectedMode === Mode.DataExplorer" @click="updateMode(Mode.DataExplorer)">
-					<IconSearchLocate32 />
-				</li>
-			</ul>
-			<ul>
-				<li
-					:active="selectedMode === Mode.ProvenanceGraph"
-					@click="updateMode(Mode.ProvenanceGraph)"
-				>
-					<IconProvenanceGraph32 />
-				</li>
-				<li :active="selectedMode === Mode.Profile" @click="updateMode(Mode.Profile)">
-					<IconUser32 />
-				</li>
-			</ul>
-		</nav>
-		<div v-if="!isCollapsed" class="mode-configuration" :class="{ right: isSidebarPositionRight }">
-			<header>{{ selectedMode }}</header>
-			<div v-if="selectedMode === Mode.Profile">
-				<Button @click="moveSidebar"> Move sidebar </Button>
-				<Button @click="logout"
-					>Logout
-					<IconLogout16 />
-				</Button>
-			</div>
-		</div>
-	</section>
+	<nav :class="{ right: isSidebarPositionRight }">
+		<ul>
+			<li :active="selectedMode === Mode.SimulationPlan" @click="updateMode(Mode.SimulationPlan)">
+				<IconDataPlayer32 />
+			</li>
+			<li :active="selectedMode === Mode.ModelView" @click="updateMode(Mode.ModelView)">
+				<IconMachineLearningModel32 />
+			</li>
+			<li :active="selectedMode === Mode.Datasets" @click="updateMode(Mode.Datasets)">
+				<IconTableSplit32 />
+			</li>
+		</ul>
+		<ul>
+			<li :active="selectedMode === Mode.ProvenanceGraph" @click="updateMode(Mode.ProvenanceGraph)">
+				<IconProvenanceGraph32 />
+			</li>
+			<li :active="selectedMode === Mode.Profile" @click="updateMode(Mode.Profile)">
+				<IconUser32 />
+			</li>
+		</ul>
+	</nav>
+	<aside v-if="!isCollapsed" class="mode-configuration" :class="{ right: isSidebarPositionRight }">
+		<header>{{ selectedMode }}</header>
+		<main v-if="selectedMode === Mode.Profile">
+			<Button @click="moveSidebar"> Move sidebar </Button>
```

Comment:
The move sidebar button is broken now. Clicking it toggles the class, but the class has no effect.